packages/php/email-editor/README.md

```diff
@@ -28,16 +23,43 @@ Example:
 composer run integration-test -- tests/integration/Engine/Theme_Controller_Test.php
 ```
 
+## Development
+
+Both **PHP** **and** JS codes are divided into `engine` and `integrations` subdirectories.
+Engine consist of code for the editor core and integrations are for extending the functionality.
+Anything **MailPoet** specific should be in the `mailpoet/lib/EmailEditor/Integrations` folder.
+
+Please avoid using MailPoet-specific services and modules in the Email editor package.
+
+### Renderer
+#### Content Renderer
+* Responsible for rendering saved template + email content to HTML or email clients
+* Flow is Preprocessors > BlocksRenderer > Postprocessors
+
+#### Root Renderer.php
+* Takes the rendered content html and places it into email HTML template template-canvas.php (We have too many items we call “template” I know 🙁)
+
+### Integrations
+Please locate integrations [here](https://github.com/mailpoet/mailpoet/tree/13bf305aeb29bbadd0695ee02a3735e62cc4f21f/mailpoet/lib/EmailEditor/Integrations/MailPoet)
+
+
 ## Known rendering issues
 
 - In some (not all) Outlook versions the width of columns is not respected. The columns will be rendered with the full width.
+- The editor may not start if the URL starts with `wp-admin/post-new.php?post_type=mailpoet_email`.
+- Most of the email editor logic heavily depends on the `mailpoet_email` post-type. This is a known issue and we are working to fix it.
+- We use `mailpoet_data` in some section of the codebase. This will be updated.
+- Native email editor implementation for `preview_url`
+- Fix the use of MailPoet vendor packages in the Email editor. We currently use `Emogrifier` and `Html2Text`.
+- Fix the Send button (button to send email) or redirect it elsewhere. It is currently pointing to the `mailpoet-newsletter` page.
+- We currently support post editing context (a post has to be created before we can use the editor). We need to add support for creation context.
```

Comment:
I would put these into a new TODO section. They are not related to rendering.

Reply:
Noted. Fixed in https://github.com/mailpoet/mailpoet/pull/5959/commits/dd8bc0d1aec393585b4e44e7826854fb93ad9b3b